gmsh_utils/gmsh_IO.py

```diff
@@ -154,103 +155,173 @@ def create_surface(self, line_ids: Union[List[int], npt.NDArray[np.int_]], name_
+            point_id = self.create_point(point, mesh_size)
+            list_point_ids.append(point_id)
+        return list_point_ids
 
-    def make_geometry_2D(self, point_coordinates: Union[List[List[float]], npt.NDArray[np.float64]],
-                         point_pairs: Union[List[List[int]], npt.NDArray[np.int_]],
-                         element_size: float, name_label: str) -> int:
+    def make_lines(self, point_pairs: Union[List[List[int]], npt.NDArray[np.int_]]) \
+            -> Union[List[int], npt.NDArray[np.int_]]:
         """
-        Takes point_pairs and puts their tags as the beginning and end of line in gmsh to create line,
-        then creates surface to make 2D geometry.
+        Makes lines with line tags by getting point pairs.
 
         Args:
-            point_coordinates (Union[List[float], npt.NDArray[np.float64]]): A list of point coordinates.
-            point_pairs (Union[List[List[int]], npt.NDArray[npt.NDArray[int]]]): A list of point tags of two
-                consecutive points in an array.
-            element_size (float): The mesh size provided by user.
-            name_label (str): The surface name label provided by user input.
+            point_pairs (Union[List[List[int]], npt.NDArray[np.int_]]): A list of pairs of point tags which create a
+            line.
 
         Returns:
-            int: Surface id.
+            List[List[int]]: A list of line tags.
         """
 
-        for point in point_coordinates:
-            coordinate = [point[0], point[1], point[2]]
-            self.create_point(coordinate, element_size)
+        line_ids = [self.create_line(point_pair ) for point_pair in point_pairs]
+        return line_ids
+
+    def make_surface(self, line_list: Union[List[int], npt.NDArray[np.int_]], name_label: str) \
+            -> int:
+        """
+        Makes surfaces with surface tags by getting line tags.
 
-        line_lists = []
+        Args:
+            line_list (Union[List[int], npt.NDArray[np.int_]]): A list of line tags in order.
+            name_label (str): surface name labels provided by user input.
 
-        for i in range(len(point_pairs)):
-            line = [point_pairs[i][0], point_pairs[i][1]]
-            line_lists.append(i + 1)
-            self.create_line(line)
+        Returns:
+            int: A list of surface tags.
+        """
 
-        surface_id: int = self.create_surface(line_lists, name_label)
-        return surface_id
+        surface = self.create_surface(line_list, name_label)
+
+        return surface
 
-    def make_geometry_3D(self, point_coordinates: Union[List[List[float]], npt.NDArray[np.float64]],
-                         point_pairs: Union[List[List[int]], npt.NDArray[np.int_]],
-                         mesh_size: float, extrusion_length: Union[List[float],
-                                                                   npt.NDArray[np.float64]], name_label: str) -> None:
+    def make_volume(self, surface_id: int,
```

Comment:
this function can be removed as it does the same as self.create_volume_by_extruding_surface(surface_id, extrusion_length, name_label)